src/plugins/decor/index.tsx

```diff
@@ -0,0 +1,225 @@
+        {
+            find: "DefaultCustomizationSections",
+            replacement: {
+                match: /{user:\i},"decoration"\),/,
+                replace: "$&$self.DecorSection(),"
+            }
+        },
+        // Obtain CustomizationSection component
+        {
+            find: ".customizationSectionBackground",
+            replacement: {
+                match: /function (\i)\(\i\){var \i,\i=\i\.title/,
+                replace: "$self.CustomizationSection=$1;$&"
+            }
+        },
+        // Decoration modal module
+        {
+            find: ".decorationGridItem",
+            replacement: [{
+                match: /,((\i)=function\(\i\){var \i=\i\.children)/,
+                replace: ";var $2;$self.DecorationGridItem=$1"
+            },
+            {
+                match: /const (\i)=(function\(\i\){var \i=\i\.user,\i=\i\.avatarDecorationOverride,\i=\i\.className)/,
+                replace: "let $1;$1=$self.AvatarDecorationPreview=$2"
+            },
+            {
+                match: /,(\i)=(function\(\i\){var \i=\i\.user,\i=\i\.avatarDecoration,)/,
+                replace: ";var $1;$self.DecorationGridDecoration=$1=$2"
+            },
+            {
+                match: /\i\.\i\.isItemViewed\((\i)\)/,
+                replace: "($1.skuId !== $self.SKU_ID ? $& : true)"
+            },
+            {
+                match: /(?<=(\i)\.label\}\),)(\i===\i\.PURCHASE\|\|\i===\i\.PREMIUM_PURCHASE&&\i)/,
+                replace: "($1.skuId === $self.SKU_ID || ($2))"
+            }]
+        },
+        {
+            find: "GUILD_STICKER_SETTINGS_REMAINING_SLOTS_AVAILABLE.format",
+            replacement: {
+                match: /(?<=numTotal:.{1,50}?,)(\i)=(function\(\i\){var \i=\i\.guildId)/,
+                replace: "$1=$self.openCreateStickerModalLazy=$2"
+            }
+        }
+    ],
+
+    flux: {
+        CONNECTION_OPEN: () => {
+            useAuthorizationStore.getState().init();
+            useCurrentUserDecorationsStore.getState().clear();
+            useUsersDecorationsStore.getState().fetch(UserStore.getCurrentUser().id, true);
+        },
+        USER_PROFILE_MODAL_OPEN: data => {
+            useUsersDecorationsStore.getState().fetch(data.userId, true);
+        },
+        MESSAGE_CREATE: data => {
+            const channel = getCurrentChannel();
+            if (channel && data.channelId === channel.id) {
+                useUsersDecorationsStore.getState().fetch(data.message.author.id);
+            }
+        },
+        TYPING_START: data => {
+            const channel = getCurrentChannel();
+            if (channel && data.channelId === channel.id) {
+                useUsersDecorationsStore.getState().fetch(data.userId);
+            }
+        },
+        LOAD_MESSAGES_SUCCESS: ({ messages }) => {
+            useUsersDecorationsStore.getState().fetchMany(messages.map(m => m.author.id));
+        },
+        // Still need to fetch for member list
```

Comment:
```suggestion
        // TODO: Still need to fetch for member list
```
(or just complete it?)